src/controllers/filters/lang/Tokenizer.hpp

```diff
@@ -25,6 +25,7 @@ static const QMap<QString, QString> validIdentifiersMap = {
     {"flags.reward_message", "channel point reward message?"},
     {"flags.first_message", "first message?"},
     {"flags.elevated_message", "elevated message?"},
+    {"flags.hype_chat", "elevated message?"},
```

Comment:
Shouldn't this be `hype chat message?` (and the `flags.elevated_message` as well)?

Reply:
probably, I just figured it copied the existing information

Reply:
Ideally the values should be unique, but I don't really see a nice way here. Since currently, the decision whether `flags.hype_chat` or `flags.elevated_message` is used in `ChannelFilterEditorDialog::ValueSpecifier::expressionText` depends on `QMap::key` (`std::map` layout in Qt 6 and internal implementation in Qt 5).

Reply:
@Felanbird if you could add a comment mentioning ^ above the hype_chat entry that'd be nice